nix/docker.nix

```diff
@@ -68,14 +62,16 @@ in
     fromImage = baseImage;
     contents = [
       cardano-node
-      startScript
     ];
     created = "now";   # Set creation date to build time. Breaks reproducibility
     extraCommands = ''
       mkdir -p data config
     '';
     config = {
-      EntryPoint = [ "start-cardano-node" ];
+      EntryPoint = [ "${cardano-node}/bin/cardano-node" "--config"
```

Comment:
@rvl this is the intended design, as per the prototype https://github.com/input-output-hk/cardano-docker-stack/blob/master/services/node/Dockerfile#L19